web/public/js/controllers/manage-dojo-users-controller.js

```diff
@@ -60,11 +62,237 @@ function cdManageDojoUsersCtrl($scope, $state, $q, cdDojoService, alertService,
+
+                  var userDojoLink = _.find(usersDojosLink, {userId:user.id});
+                  if(!userDojoLink.userTypes) userDojoLink.userTypes = [];
+                  var roles = [];
+                  if (age < 13) {
+                    roles.push('attendee-u13');
+                  } else if (age < 18) {
+                    roles.push('attendee-o13');
+                  }
+                  if (userType && userType.name) {
+                    roles.push(userType.name);
+                  }
+                  $scope.selectedItems[0].subCaption = userUtils.getTitleForUserTypes(roles, user);
+                  userDojoLink.userTypes = roles;
+                  cdDojoService.saveUsersDojos(userDojoLink, function (response) {
+                    if(response.error) {
+                      reject($translate.instant(response.error));
+                    } else {
+                      resolve($translate.instant('User types successfully updated.'));
+                    }
+                  }, function (err) {
+                    reject($translate.instant('Error saving user type') + ' ' + err);
+                  });
+                } else {
+                  reject($translate.instant('You do not have permission to update user types'));
+                }
+              });
+            });
+          },
+          title: $translate.instant('Change Role')
+        });
+      }
+    },
+    remove: {
+      ngClick: function () {
+        var user = $scope.selectedItems[0].userData;
+        $ngBootbox.confirm($translate.instant('Are you sure you want to remove this user from your Dojo?'))
+          .then(function() {
+            $scope.removeUser(user);
+          });
+      },
+      ngShow: function () {
+        return true;
+      }
+    },
+    backgroundCheck: {
+      model: function () {
+        var user = $scope.selectedItems.length > 0 ? $scope.selectedItems[0].userData : null;
+        return (user && user.backgroundChecked);
+      },
+      ngShow: function () {
+        var user = $scope.selectedItems.length > 0 ? $scope.selectedItems[0].userData : null;
+        return (user && user.isMentor && $scope.canUpdateUserPermissions);
+      },
+      onChange: function (checked) {
+        var user = $scope.selectedItems.length > 0 ? $scope.selectedItems[0].userData : null;
+        if (user) {
+          user.backgroundChecked = checked;
+          $scope.updateMentorBackgroundCheck(user);
+        }
+      }
+    }
+  };
+
+  function updateActions() {
+    $scope.userPermActions = [];
+    $scope.userPermissions.forEach(function (userPermission) {
+      $scope.userPermActions.push({
+        type: 'checkbox',
+        title: userPermission.title,
+        model: function () {
+          var users = $scope.selectedItems;
+          if (users.length === 1) {
+            var user = users[0].userData;
+            return $scope.userPermissionsModel[user.id][userPermission.name];
+          }
```

Comment:
Default would just be undefined or null, seems pointless when undefined would be default.